frontend/src/components/atoms/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Container = styled.div`
+  padding: 0;
+  margin: auto 10px;
```

Comment:
Skąd się właściwie wzięło to 10px? Jeżeli dobrze umiem to odczytać, to na Figma jest chyba 16px. 
Generalnie zostawiłbym tutaj `0`, żeby w sytuacji jak trzeba będzie z tego elementu skorzystać gdzieś indziej, to nie będziemy musieli korzystać z ujemnych marginesów, jeżeli trzeba byłoby element przemieścić

Reply:
poprawione